app/src/main/res/layout-land/fragment_onboarding.xml

```diff
@@ -78,4 +79,183 @@
+              android:layout_width="64dp"
+              android:layout_height="64dp"
+              app:layout_constraintTop_toTopOf="parent"
+              app:layout_constraintBottom_toBottomOf="parent"
+              app:layout_constraintStart_toStartOf="parent"
+              app:layout_constraintEnd_toEndOf="parent"
+              app:lottie_autoPlay="true"
+              app:lottie_enableMergePathsForKitKatAndAbove="true"
+              app:lottie_loop="true"
+              app:lottie_rawRes="@raw/three_dots_loading"
+              android:visibility="visible"
+              tools:visibility="visible"
+              />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+      </com.google.android.material.card.MaterialCardView>
+
+      <TextView
+          android:id="@+id/onboarding_recover_text4"
+          style="@style/OnboardingValuesBodyTextStyle"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="8dp"
+          android:gravity="center"
+          android:text="@string/intro_from_web_payment_to_spend_body"
+          app:layout_constraintBottom_toBottomOf="@+id/onboarding_recover_guest_button"
+          app:layout_constraintEnd_toStartOf="@+id/onboarding_recover_guest_button"
+          app:layout_constraintHorizontal_bias="0.03"
+          app:layout_constraintStart_toEndOf="@+id/onboarding_bonus_layout"
+          app:layout_constraintTop_toTopOf="@+id/onboarding_recover_guest_button"
+          />
+
+      <com.appcoins.wallet.ui.widgets.WalletButtonView
+          android:id="@+id/onboarding_recover_guest_button"
+          android:layout_width="198dp"
+          android:layout_height="@dimen/onboarding_button_height"
+          android:layout_marginEnd="16dp"
+          app:buttonText="@string/intro_get_started_button"
+          app:buttonType="filled"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="parent"
+          />
+
+      <com.airbnb.lottie.LottieAnimationView
+          android:id="@+id/loading_animation"
+          android:layout_width="80dp"
+          android:layout_height="80dp"
+          android:visibility="invisible"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintStart_toStartOf="@+id/onboarding_recover_guest_button"
+          app:lottie_autoPlay="true"
+          app:lottie_enableMergePathsForKitKatAndAbove="true"
+          app:lottie_loop="true"
+          app:lottie_rawRes="@raw/loading_wallet"
+          tools:visibility="visible"
+          />
+
+      <TextView
+          android:id="@+id/onboarding_recover_text5"
+          style="@style/OnboardingRecoverGuestBodyTextStyle"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="28dp"
```

Comment:
material design works in a 8dp grid, please avoid using non multiple of 8. https://m2.material.io/design/layout/spacing-methods.html#baseline-grid